### ISSUE TITLE
`prevent-abbreviations` default option is not good

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#271]
`prevent-abbreviations` is really good, I have tried to lint my old scripts.

after a few test, I notice that

`checkDefaultAndNamespaceImports` default value is `false`, but normally change the name of `default and namespace` import has no side effect, this should set to `true`, I think.

`checkProperties` default value is `true`, but normally when export or return a object, the prop name can't just rename without api changes, I think this should set to `false` by default.

**Comment by fisker** [comment]
This is what I'm using, feel better when linting, no need change api (including internal using props)

```js
    'unicorn/prevent-abbreviations': [
      'error',
      {
        checkDefaultAndNamespaceImports: true,
        checkProperties: false,
      },
    ],
```

**Comment by futpib** [comment]
Could you please split this into two separate issues (one for each default we've chosen)?

**Comment by futpib** [comment]
Some reasons behind `checkDefaultAndNamespaceImports: true` are outlined in the discussion starting from here https://github.com/sindresorhus/eslint-plugin-unicorn/pull/237#issuecomment-470942156

Regarding `checkProperties`, we currently ignore names exported via `export` keyword. Maybe we should ignore `module.exports` too.

**Comment by fisker** [comment]
thanks for explaining `checkDefaultAndNamespaceImports`.

for `checkProperties`, I don't think `export` / `module.exports` is good enough, how do you know `return` is not a kind of "exporting", maybe just for another function to use, or handwriting `umd` or `iife` mod. 

since those two options can easily overwrite, I'm going to keep my own choices.

**Comment by sobolevn** [comment]
I am facing this issue too. Since the new release linter checks options that I do not have control over. That's sad. 

Failing build: https://travis-ci.org/wemake-services/wemake-vue-template/builds/513997750
Code: https://github.com/wemake-services/wemake-vue-template/blob/master/template/nuxt.config.ts#L93

This is a default `Nuxt` function which works with the default API:

```js
{
  'build': {
      // Error here on `isDev`:
      extend (config, { isDev, isClient }) { ... }
  }
}
```

**Comment by fisker** [comment]
https://github.com/xojs/xo/blob/c55f962a961b5b00b4b4239a7253a2e874b0e700/config/plugins.js#L40

**Comment by sindresorhus** [comment]
> Regarding checkProperties, we currently ignore names exported via export keyword. Maybe we should ignore module.exports too.

👍

**Comment by sindresorhus** [comment]
> for checkProperties, I don't think export / module.exports is good enough, how do you know return is not a kind of "exporting", maybe just for another function to use, or handwriting umd or iife mod.

@fisker Can you provide some code examples of this?

**Comment by sindresorhus** [comment]
I think we should make `checkProperties` false by default for now.

**Comment by sindresorhus** [comment]
@sobolevn That's weird. According to our docs, [`checkShorthandProperties`](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/prevent-abbreviations.md#checkshorthandproperties) is false by default.
